data/themes/darktable.css

```diff
@@ -1596,15 +1596,16 @@ margin: 0 2px;
   margin-left: 0.5em;
 }
 
-#modulegroups-popup *
+#modulegroups-popup-item
 {
- padding-top: 0.1em;
- padding-bottom: 0.1em;
+  padding-left: 2em;
+  padding-top: 0.1em;
+  padding-bottom: 0.1em;
 }
 
-#modulegroups-popup-title *
+#modulegroups-popup-title
 {
-  padding-left: 2em;
+  padding-left: 0em;
   padding-bottom: 0.2em;
```

Comment:
@AlicVB: I've just tested more that in master. It's all good. Just a thing: I find that title need more space on top of them, just to aerate a little. It's just too tight as it is between title and modules, so I would add a padding-top of 0.4em. As you will work on a PR to make submenus, could you remove lines 1608 and 1609 above and replace them to that:
`
padding: 0.4em 0 0.2em 0;`

@elstoc: hope this little change is ok for you?

Reply:
@Nilvus : no problem. I just need to remember :)